tests/marketplace/deployer_helm_tiller_base/onbuild/standard/chart/minimal-helm-chart/values.yaml

```diff
@@ -0,0 +1,3 @@
+tls:
```

Comment:
what's the reason for expliciting specifying nulls here?

Reply:
Having the values into the `values.yaml` file is not required, however it seems to be a good practice to maintain a full working helm chart, so the end-user can use `--set` flags to provide the certificate during the manifest is expanding.